src/nebula-hooks/virtualized-table/use-wait-for-fonts.ts

```diff
@@ -0,0 +1,27 @@
+import { usePromise } from '@nebula.js/stardust';
+import { getBodyStyle, getHeaderStyle } from '../../table/utils/styling-utils';
+import { ExtendedTheme, TableLayout } from '../../types';
+
+const useWaitForFonts = (theme: ExtendedTheme, layout: TableLayout) => {
+  const { fontSize: headerFontSize, fontFamily: headerFontFamily } = getHeaderStyle(layout, theme, false);
```

Comment:
Could consider to convert the `useTableStyling` hook into a Nebula hook, so we don't need to resolve the styling twice. But it's not a costly operation so not really sure it's worth it.